Jobs/Metadata/MetadataChangedJob.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file Jobs/Metadata/MetadataChangedJob.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class MetadataChangedJob
+ * @ingroup jobs
+ *
+ * @brief Classe to handle the Metadata Changed job
+ */
+
+namespace PKP\Jobs\Metadata;
+
+use APP\core\Application;
+use APP\core\Services;
+use PKP\config\Config;
+use PKP\Domains\Jobs\Exceptions\JobException;
+
+use PKP\Support\Jobs\BaseJob;
+
+class MetadataChangedJob extends BaseJob
+{
+    /**
+     * The name of the connection the job should be sent to.
+     *
+     * @var string|null
+     */
+    public $connection;
+
+    /**
+     * @var int The submission ID
+     */
+    protected $submissionId;
+
+    /**
+     * Create a new job instance.
+     *
+     */
+    public function __construct(int $submissionId)
+    {
+        $this->connection = Config::getVar('queues', 'default_connection', 'sync');
```

Comment:
I think you're trying to provide for use of both `sync` and `database` queues, but the `database` queue should always be available, and I think we'll be dependent enough on it that we should focus on making it work in all deployments (e.g. shutdown function as well as cron and daemon-based approaches) rather than planning to fall back on `sync`.